ddp-workspace/projects/ddp-dsm-ui/src/app/shipping-search/shipping-search.component.ts

```diff
@@ -127,6 +128,9 @@ export class ShippingSearchComponent implements OnInit {
     if (typeof value === 'string') {
       kitRequest[ parameterName ] = value;
       v = value;
+      if(parameterName == "collectionDate"){
```

Comment:
the field wasn't saved at this point, so the message could say it was saved but there could still be a server error that prevented saving

Reply:
Where does it write to the server? Here? `this.patch( patch );`

Reply:
@ngigliotosBroad yes, we are sure it got saved at L 154

Reply:
Alright, it now only saves on a successful subscribe call, so if the server write files the message should not show. It will now also hide the message if someone tries to change the date again after an initial change.